keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterHandler.kt

```diff
@@ -192,12 +195,22 @@ class TitusClusterHandler(
       }.toSet()
     )
 
+    val deployStrategy = clusterExportHelper.discoverDeploymentStrategy(
+      cloudProvider = "titus",
+      account = exportable.account,
+      application = exportable.moniker.app,
+      serverGroupName = base.name
+    )
+      ?.withDefaultsOmitted()
+      ?: RedBlack().withDefaultsOmitted()
+
```

Comment:
Nit: this reads kind of weird to me - i'd probably remove both the `.withDefaultsOmitted()` calls here (and in the ec2 cluster handler) and then just put it on the spec call below (so here line 213 would have `deployWith = deployStrategy.withDefaultsOmitted()`). I think that makes it a bit easier to read, because I don't have to reason about when `.withDefaultsOmitted()` would be null, then realize that's only the case when actually the `deployStrategy` was null.